gammapy/data/tests/test_observations.py

```diff
@@ -37,6 +37,25 @@ def test_data_store_observation(data_store):
     assert_skycoord_allclose(obs.target_radec, c)
 
 
+@requires_data('gammapy-extra')
+def test_data_store_observation_to_observation_cta(data_store):
+    from gammapy.data import EventList, GTI, ObservationCTA
+    from gammapy.irf import EffectiveAreaTable2D, EnergyDispersion2D, EnergyDependentMultiGaussPSF
```

Comment:
Move imports to top of file, use relative imports within gammapy

Reply:
Done